webapp/src/components/properties/link/link.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ReactNode} from 'react'
+
+import Editable from '../../../widgets/editable'
+
+import './link.scss'
+import {Utils} from '../../../utils'
+
+type Props = {
+    value: string
+    onChange: (value: string) => void
+    onSave: () => void
+    onCancel: () => void
+    validator: (newValue: string) => boolean
+}
+
+const URLProperty = (props: Props): JSX.Element => {
+    let link: ReactNode = null
+    if (props.value?.trim()) {
+        link = (
+            <a
+                className='icon-link-variant'
+                href={Utils.ensureProtocol(props.value.trim())}
+                target='_blank'
+                rel='noreferrer'
+            />
+        )
+    }
+
+    return (
+        <div className='property-link url'>
```

Comment:
We use by convetion that the className of the root div of a component is the component name, in this case is `URLProperty`, that way uppercased classes are clearly distinguished as component classes. and also is easy to correlate them to the correct react component.

Reply:
Got it. Done.